py3status/modules/process_status.py

```diff
@@ -21,14 +27,14 @@
 import os
 import subprocess
 
-ERR_NO_PROCESS = 'no process name given'
-
 
 class Py3status:
     """
     """
     # available configuration parameters
     cache_timeout = 10
+    format = '{state}'
+    format_error = 'no process name given'
```

Comment:
no anything but format in my view.

This is an error message not a format it should never see the formatter etc.  Anyhow I think this should be discussed elsewhere as it may block this PR until a decision is made.  So I'd actually undo that change.